superset/assets/src/visualizations/big_number.css

```diff
@@ -1,31 +1,40 @@
-.big_number g.axis text,
-.big_number_total g.axis text {
-  font-size: 10px;
-  font-weight: normal;
-  color: gray;
-  fill: gray;
-  text-anchor: middle;
-  alignment-baseline: middle;
-  font-weight: none;
+.big_number {
+  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Open Sans,Helvetica Neue,sans-serif;
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
 }
 
-.big_number text.big,
-.big_number_total text.big{
-  stroke: black;
-  text-anchor: middle;
-  fill: black;
+.big_number.no_trendline .big_number_subheader_line {
+  padding-bottom: 0.3em;
 }
 
-.big_number g.tick line,
-.big_number_total g.tick line{
-  stroke-width: 1px;
-  stroke: grey;
+.big_number_text_container {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
 }
 
-.big_number .domain,
-.big_number_total .domain{
-  fill: none;
-  stroke: black;
-  stroke-width: 1;
+.big_number_header_line {
+  position: relative;
+  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Open Sans,Helvetica Neue,sans-serif;
```

Comment:
do you need to define `font-family` more than once or can you inherit from the container? or is this necessary for proper size-detection?

Reply:
Necessary for proper size detection. Otherwise the util function will have to create same nesting structure (with outer container have `.big_number`) or be able to accept parent DOM node (which does not exist yet).

I understand this looks annoyingly redundant, but don't have a good work around yet. Unless all body use same font family.

Reply:
makes sense 👍and I agree no obvious work around. We could possibly add a comment in the css that removing it would potentially break proper sizing.

Reply:
I thought about this again and made the util function accept container DOM element, therefore got rid of the font family.